web-registry/src/pages/ProjectReview/ProjectReview.VCSMetadata.tsx

```diff
@@ -43,8 +43,12 @@ const VCSMetadata: React.FC<Props> = ({ metadata }) => {
       </ItemDisplay>
       <ItemDisplay name="Project start and end date">{`${formatDate(
         metadata?.['regen:projectStartDate']?.['@value'],
+        undefined,
```

Comment:
why do we pass an `undefined` here?

Reply:
there probably is a better way but i'm really not feeling to strongly about it. it just reads a bit funny.